app/scripts/filters/resources.js

```diff
@@ -1038,7 +1038,8 @@ angular.module('openshiftConsole')
             signal,
             exitCode;
 
-        if (containerReason) {
+        // only show containerReason if pod.status.phase is not 'Failed'
+        if (reason !== 'Failed' && containerReason) {
```

Comment:
@spadgett can you think of a case where this isn't what we would want?

Reply:
I feel like there might be specific reasons you'd want to see even for failed pods. Evicted? Deadline exceeded?

This logic was modeled after the CLI, so it would be good to stay consistent. I'm curious what the CLI shows for these pods.

Reply:
```
NAME                READY     STATUS       RESTARTS   AGE
dancer-ex-1-build   0/1       Init:Error   0          1h
```

Reply:
@jwforres Do we need to iterate over `initContainerStatuses` here too?

Reply:
I believe this is the current CLI logic if we want to be consistent:

https://github.com/spadgett/origin/blob/5b71e8d8e441a0d39f9228a4bce4c7a6a13be08b/vendor/k8s.io/kubernetes/pkg/printers/internalversion/printers.go#L523-L615

Reply:
yeah i think that is probably the correct fix for this

Reply:
Looks like CLI has special initializing logic.